wrangler-core/src/main/java/io/cdap/directives/column/Copy.java

```diff
@@ -110,4 +112,18 @@ public Mutation lineage() {
       .conditional(source.value(), destination.value())
       .build();
   }
+
+  @Override
+  public Schema getOutputSchema(SchemaResolutionContext context) {
+    Schema inputSchema = context.getInputSchema();
+    return Schema.recordOf(
+      "outputSchema",
+      inputSchema.getFields().stream()
+        .map(
+          field -> field.getName().equals(destination.value()) ?
```

Comment:
If the destination column does not exist in the input schema, new column is not added to output schema.